types/yeoman-environment/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for yeoman-environment 2.10
+// Type definitions for yeoman-environment 3.4.1
```

Comment:
Please revert this change - I don't think it's necessary as you didn't edit the `yeoman-environment`-typings

Reply:
If you, for some reason, would like to bump the version of `@types/yeoman-environment` you have to change the version-number to `3.4`, but I don't think it's necessary, tho